addon/components/polaris-form.js

```diff
@@ -0,0 +1,132 @@
+import Component from '@ember/component';
+import layout from '../templates/components/polaris-form';
+import { normalizeAutoCompleteProperty } from '../utils/normalize-auto-complete';
+
+export default Component.extend({
+  tagName: 'form',
+  attributeBindings: [
+    'acceptCharset',
+    'action',
+    'autoCompleteInputs:autoComplete',
```

Comment:
Does this need to be `autocomplete` rather than `autoComplete`? 🤔

Reply:
good catch...this should be `autocomplete`